content/news/2021-02-14-week-11-progress.md

```diff
@@ -0,0 +1,15 @@
+title: "About my Outreachy internship with Mixxx DJ: Week 11 progress"
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship
+date: 2021-02-14 12:09:00
+
+_This is the eleventh weekly progress report of my Outreachy internship with Mixxx DJ._
+
+This week:
+
+- I successfully added [rekordbox](https://rekordbox.com/en/) (and its icon) to the list of supported libraries in this [pull request](https://github.com/mixxxdj/manual/pull/322) that I opened a long while ago – I was documenting the rekordbox library import.
+- I also started a LinkedIn company page for Mixxx, on the suggestion of my mentor. I think it will be a great way to get Mixxx to reach an even wider audience. Here is the [link](https://www.linkedin.com/company/mixxx-dj/) to the page.
+- I learned something new this week! I learned how to use [rsync](https://linux.die.net/man/1/rsynchttps://linux.die.net/man/1/rsync) and [OpenSSH](https://www.openssh.com/) inside the [cygwin](https://www.cygwin.com/) command-line to send files from one computer to another. The learning curve was slightly steep but I finally got the hang of it. I sent the YouTube Mixxx tutorials that I have made so far through the cygwin command-line to my mentor, Owen.
+- I added more description to the section in the manual that talks about analyzing tracks/ the music library. The manual did not define what analysis does and why it is important, so I felt like it could easily be overlooked – especially by the new user. I added some information about why users should do it in advance and what happens if they don’t. I opened a pull request [here](https://github.com/mixxxdj/manual/pull/351#event-4327068278).
```

Comment:
```suggestion
- I added more documentation to the track/library analysis section in the manual. The manual did not define what analysis does and why it is important, so I felt like it could easily be overlooked – especially by new users. I added some information about why users should do it in advance and what happens if they don’t. I opened a pull request [here](https://github.com/mixxxdj/manual/pull/351).
```